src/script/ismine.cpp

```diff
@@ -13,6 +13,12 @@
 
 typedef std::vector<unsigned char> valtype;
 
+enum class IsMineSigVersion
```

Comment:
commit: Switch to a private version of SigVersion inside IsMine

Please let a comment explaining what this is and how it differs from SigVersion. Even though it's in the commit comment, it's good to have it in the code.

Reply:
Done (in the next commit, which actually introduces the difference).